src/tools/rbd/action/MirrorImage.cc

```diff
@@ -292,15 +307,24 @@ int execute_status(const po::variables_map &vm,
     formatter->dump_string("global_id", status.info.global_id);
     formatter->dump_string("state", state);
     formatter->dump_string("description", status.description);
+    formatter->open_object_section("daemon_service");
```

Comment:
Nit: move this into ```daemon_service_info.dump```?

Reply:
I like the idea of dumping the empty object section, which I think could still occur if you just always invoke the ```dump``` and tweak the current code that immediately returns if the instance wasn't found.